src/components/Comment/Comments.scss

```diff
@@ -0,0 +1,74 @@
+@import "~styles/main";
+
+.comment {
+  @include font(Poppins);
+  margin: 20px auto;
+  width: 60%;
+
+  //shame: this is very hacky method. If found an alternative, please replace
+  .avatar {
+    position: fixed;
+    margin: 0px -60px;
+  }
+
+  &__top {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    margin: 0 0 5px 0;
+    padding: 0;
+    .comment__author {
+      font-weight: 700;
+      font-size: 0.75rem;
+      display: inline-block;
+      vertical-align: middle;
+      // margin-right: 200px;   // shame- cant give margin right of text
```

Comment:
if shame is resolved, it's better to remove it rather than commenting it.
Or if you want it as a comment, please give a description so that someone from the future will not get confused